js/binance.js

```diff
@@ -4894,4 +4894,47 @@ module.exports = class binance extends Exchange {
     async addMargin (symbol, amount, params = {}) {
         return await this.modifyMarginHelper (symbol, amount, 1, params);
     }
+
+    async getMarginInterestRate (currency, since = undefined, limit = undefined, tier = undefined, params = {}) {
```

Comment:
I would suggest to prepend all info-fetching methods with `fetch` instead of `get`.